initializers/sequelize.js

```diff
@@ -60,7 +68,8 @@ module.exports = {
         fs.readdirSync(dir).forEach(function(file){
           var nameParts = file.split("/");
           var name = nameParts[(nameParts.length - 1)].split(".")[0];
-          api.models[name] = api.sequelize.sequelize.import(dir + '/' + file);
+          var modelFunc = currySchemaFunc(require(dir + '/' + file));
+          api.models[name] = api.sequelize.sequelize.import(capitalizeFirstLetter(name), modelFunc);
```

Comment:
So.. I'm not confident that capitalizing the first letter of the model name cuts it. Can anyone provide input?